src/x/xsystem.c

```diff
@@ -354,6 +360,31 @@ static bool xglx_get_cursor_position(int *ret_x, int *ret_y)
 static bool xglx_inhibit_screensaver(bool inhibit)
 {
    ALLEGRO_SYSTEM_XGLX *system = (void *)al_get_system_driver();
+   int temp, temp_version_min, temp_version_max;
```

Comment:
It makes no sense to name the last two variables `temp_`. Just call them `version_min`, `version_max`.